tests/Resources/ti.geolocation.test.js

```diff
@@ -680,7 +680,7 @@ describe('Titanium.Geolocation', () => {
 					should(data.places[0]).have.property('latitude').which.is.a.Number();
 					should(data.places[0]).have.property('longitude').which.is.a.Number();
 					should(data.places[0].country).be.oneOf('USA', 'United States of America', 'United States');
-					should(data.places[0].state).be.eql('California');
+					should(data.places[0].state).be.oneOf('California', 'CA');
```

Comment:
Interesting. This is an example on how this is a breaking-change and where the results may differ between Android and iOS... but we've got to do it anyways.

Reply:
We're missing "state" property documentation in `GeocodedAddress` and `ForwardGeocodeResponse`. Would you  mind adding it please? We can indicate that the "state" property might return the full name or abbreviated name.
https://github.com/appcelerator/titanium_mobile/blob/master/apidoc/Titanium/Geolocation/Geolocation.yml

Reply:
Added doc for state property.